src/main/java/net/minecraftforge/gradle/tasks/dev/GeneratePatches.java

```diff
@@ -82,14 +85,9 @@ public void visitFile(FileVisitDetails f) {
         });
 
         // We want things sorted in reverse order. Do that sub folders come before parents
-        Collections.sort(directories, new Comparator<File>() {
-            @Override
-            public int compare(File o1, File o2) {
-                int r = o1.compareTo(o2);
-                if (r < 0) return 1;
-                if (r > 0) return -1;
-                return 0;
-            }
+        directories.sort((o1, o2) -> {
+            int r = o1.compareTo(o2);
+            return Integer.compare(0, r);
```

Comment:
Comparator.reversedOrder() can be good

Reply:
is it now reversed ?

Reply:
I'm sorry for my typo. `directories.sort(Comparator.reverseOrder())` is good I think